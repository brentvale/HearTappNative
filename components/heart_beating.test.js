import 'react-native';
import React from 'react';
import HeartBeating from './heart_beating';

import renderer from 'react-test-renderer';

test('renders properly', () => {
	const tree = renderer.create(
		<HeartBeating />
	).toJSON();
	expect(tree).toMatchSnapshot();
})