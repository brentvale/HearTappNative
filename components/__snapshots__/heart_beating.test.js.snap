// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<View
  style={
    Object {
      "alignItems": "flex-start",
      "flex": 1,
      "justifyContent": "flex-start",
      "paddingTop": 0,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
  />
</View>
`;
